### ISSUE TITLE
Adding a warning if remote file/folder is more recent as local before uploading.

### ISSUE THREAD

**Issue by Natizyskunk** [Natizyskunk/vscode-sftp#394]

**Comment by Aviv-rs** [comment]
Yes please!! this is so crucial, My team is using Adobe Dreamweaver because of this feature
I would really like to throw it out the window and switch to VSCode

**Comment by gianluca-github** [comment]
Please, it's very important! Many errors occurs in my team. I'm the only one who use VSCode,  the other stay on Dreamweaver, I would like  they get in on VSCode!
